software/bkeymaps.txt

```diff
@@ -0,0 +1,37 @@
+BKEYMAPS [0]
+________________________________________________________________________________
+
+Binary keymaps for busybox
```

Comment:
`loadkeys` (from `kbd`) also supports these keymap files: https://jlk.fjfi.cvut.cz/arch/manpages/man/loadkeys.1 

They're not only for busybox in other words.